src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -196,33 +225,51 @@ private void update() {
 
         root.setRedraw(false);
         
-        Set<String> selectedFeatures = model.getSelectedFeatures();
-        
         for (TableItem item : features.getItems()) {
             item.dispose();
         }
-        
-        for (int i = 0; i < handle.getNumColumns(); i++) {
-            TableItem item = new TableItem(features, SWT.NONE);
-            String value = handle.getAttributeName(i);
-            item.setText(value);
-            item.setChecked(selectedFeatures.contains(value));
-        }
-        
-        Set<String> selectedClasses = model.getSelectedClasses();
-        
         for (TableItem item : classes.getItems()) {
             item.dispose();
         }
         
         for (int i = 0; i < handle.getNumColumns(); i++) {
-            TableItem item = new TableItem(classes, SWT.NONE);
-            String value = handle.getAttributeName(i);
-            item.setText(value);
-            item.setChecked(selectedClasses.contains(value));
+            updateTableItem(handle, i, features, model.getSelectedFeatures());
+            updateTableItem(handle, i, classes, model.getSelectedClasses());
         }
         
         root.setRedraw(true);
         SWTUtil.enable(root);
     }
+    
+    /**
+     * Update table item.
+     * 
+     * @param handle
+     * @param i
+     * @param table
+     * @param set
+     */
+    private void updateTableItem(DataHandle handle, int i, DynamicTable table, Set<String> set){

```

Comment:
This  should be called createTableItem, as it does not update items but create items.